docs/run-conformance-tests.md

```diff
@@ -0,0 +1,29 @@
+# Conformance tests
+
+This document enumerates the way steps to run conformance tests for various platforms supported by Lokomotive.
+
+## Running conformance tests
+
+Follow the canonical document [here](https://github.com/cncf/k8s-conformance/blob/master/instructions.md) which instructs on installing sonobuoy and running tests.
+
+## AWS
+
+For AWS you need to make sure that the node ports are allowed in the security group. To do so run following commands which adds a snippet to the file [`security.tf`](https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/security.tf).
+
+```tf
+echo '
+resource "aws_security_group_rule" "worker-nodeport" {
```

Comment:
I think we should make this as a configuration flag, so one does not have to modify the source code.

Reply:
done.